### ISSUE TITLE
V9.2.1, build 15998, platform linux-x86_64 search files but not sub-directories

### ISSUE THREAD

**Issue by Xotic750** [Komodo/KomodoEdit#681]
When I use the the `search files` facility and set the directory I want to search, I switch off the `Search In sub-Directories` option but it still searches the sub.directories.

**Comment by Naatan** [comment]
Does it work if you check then uncheck the checkbox?

**Comment by Xotic750** [comment]
Nope, same result.

**Comment by Naatan** [comment]
Thanks, that narrows it down believe it or not ;)

**Comment by Xotic750** [comment]
You're welcome.

**Comment by Naatan** [comment]
See if this still happens in the next nightly (tomorrow).

**Comment by Xotic750** [comment]
Will try to take a look at it and let you know.

**Comment by Xotic750** [comment]
While installing
```
[graham@tyr Komodo-Edit-9.3.0-16352-linux-x86_64]$ ./install.sh 
install: warning: Your system's version of %s is not up to date.
install: warning: Komodo requires version 2.24 or greater. Please update gdk-2.0. While Komodo will still install correctly, it will not run.
```
```
[graham@tyr Komodo-Edit-9.3.0-16352-linux-x86_64]$ dnf list gtk2
Last metadata expiration check performed 0:04:30 ago on Thu Oct 22 23:24:16 2015.
Installed Packages
gtk2.i686        2.24.28-1.fc22   @System
gtk2.x86_64   2.24.28-1.fc22   @System
```
Still runs up ok though.

**Comment by Naatan** [comment]
And the file searching? :)

**Comment by Xotic750** [comment]
Just been trying that, seems to be still searching sub.directories. Oh, it's really difficult for the colour blind to see whether search sub-directories is enabled or not.

**Comment by Naatan** [comment]
I cannot reproduce this, it works fine for me :\ 

If you are having trouble telling colors apart try switching to a different color scheme that has more contrast.

**Comment by Xotic750** [comment]
I'll have al look to try to find the colour selector for that item but I have the scheme set to what is best for my colour blindness. Going to upload a video, you will also see that the icons are still not visible on this install and after deleting those directories that you suggested in your forum post.
https://www.youtube.com/watch?v=CLxx6l315_4&feature=youtu.be
Don't know what more I can do to help debug this.

**Comment by Naatan** [comment]
Are you on the latest nightly?

**Comment by Xotic750** [comment]
No, I'm still on the 9.2.1 release (daily) but I can install for testing stuff.

**Comment by Naatan** [comment]
You are most definitely not on 9.2.1 (as evident by your video) ;)

**Comment by Xotic750** [comment]
Yes, in the video I had installed the 9.3.0 nightly of that days post. When I said "daily" I meant the stable release, which I use daily, my bad ;)

**Comment by mitchell-as** [comment]
I've been able to reproduce this.

I have a folder "/tmp/test/".

I have a "/tmp/test/foo.txt" file with the contents

    foo
    bar

I have a "/tmp/test/bar/bar.txt" file with the contents

    bar

Ctrl+Shift+F, search for "bar", enter "/tmp/test" in the Directories inputbox, and unselect the "search in subdirectories" option.

The result finds 2 occurrences, including the "bar" in "bar/bar.txt", but the "bar/" subdirectory should not have been searched. I get these same two results no matter what my "search in subdirectories" option is.
